jsonparser.cpp

```diff
@@ -29,23 +30,36 @@
 #include <QVersionNumber>
 #include <algorithm>
 
-bool compareVersion(const QVariantMap &imageMap1, const QVariantMap &imageMap2)
+QTextStream& qStdOut()
```

Comment:
why doesn't simple `qDebug()` suit your needs? And you already include `QDebug`.

Reply:
It would never print anything to the console. But using this way worked.  I will remove this once the PR is ready.

Reply:
Ah, it’s because I built QT6 for release and not debug.